docker/cygnus-ngsi/Dockerfile

```diff
@@ -65,9 +67,9 @@ RUN yum -y install tar nc which git java-1.6.0-openjdk-devel && \
 # Create Cygnus log folder
 RUN mkdir ${FLUME_HOME}/log
 
-# Clone fiware-cygnus
-RUN git clone ${GIT_URL_CYGNUS} ${CYGNUS_HOME} && \
-    cd ${CYGNUS_HOME} && \
+# Instead of re-cloning, copy the fiware-cygnus repo, move there and checkout the propper branch
+COPY . ${CYGNUS_HOME}
```

Comment:
I couldn't make this work:
```bash
fatal: Not a git repository (or any of the parent directories): .git
The command '/bin/sh -c cd ${CYGNUS_HOME} &&     git checkout ${GIT_REV_CYGNUS}' returned a non-zero code: 128
```
Makes all the sense, `.` points to `fiware-cygnus/docker/cygnus-ngsi/.`, and that's not a git repository